by.teachmeskills/homework13/src/main/java/SalaryTax.java

```diff
@@ -0,0 +1,22 @@
+
+//        Вычислить сумму налога по прогрессивной шкале в зависимости от полученного заработка:
+//        13% на доход до 10 000, 20% на доход от 10 000 до 50 000, 30% на доход выше 50 000.
+//        Метод должен принимать сумму оплаты труда, а возвращать сумму налога.
+
+class SalaryTax {
```

Comment:
![image](https://user-images.githubusercontent.com/29692064/215293407-be9f7c08-69d9-4e76-99a8-69586a48d683.png)

Reply:
package is not defined

Reply:
I've fixed the structure and added Dataprovider